README.md

```diff
@@ -100,64 +34,53 @@ jobs:
           level: warning
 ```
 
-## Advanced use cases
+## Inputs
 
-This action can be combined with [peter-evans/create-pull-request](https://github.com/peter-evans/create-pull-request) or [stefanzweifel/git-auto-commit-action](https://github.com/stefanzweifel/git-auto-commit-action) to also apply the annotated changes to the repository.
+### `github_token`
 
-### Commit changes
+**Required**. Must be in form of `github_token: ${{ secrets.github_token }}`'.
```

Comment:
It's inconsistent with the description in action.yml.
It's better to clarify it's filled by default.

And the description of github_token in action.yml is not correct as I described in https://github.com/reviewdog/action-remark-lint/pull/31.

Reply:
Fixed similar to [reviewdog/action-remark-lint/pull/31](https://github.com/reviewdog/action-remark-lint/pull/31)